shakenfist/util.py

```diff
@@ -260,3 +261,22 @@ def ignore_exception(processname, e):
         msg += '\n%s' % traceback.format_exc()
 
     LOG.error(msg)
+
+
+METHOD_RE = re.compile('.* in (.*)')
+
+
+def log(level, relatedobjects, message):
```

Comment:
I think you should have a command for each log level.

Reply:
Mutter.